get_ioc_usage.py

```diff
@@ -78,6 +99,8 @@ def main():
                              "given instruments. If not defined, tests will be run on all instruments.")
     parser.add_argument("--ioc", type=str, default=None, help="If specified show instruments with IOC starting with"
                                                               "this string, otherwise show all iocs on the instruments")
+    parser.add_argument("--device_screens", help="Show if an instrument is missing an IOC",
```

Comment:
I think the help string could be more descriptive and say something like "Show if an instrument has device screens that reference a missing IOC" or even just "print IOCs in device screens for instruments"